pages/docs/frontend-modules/releasing.en-US.mdx

```diff
@@ -1,17 +1,26 @@
+import { Callout } from "nextra-theme-docs";
+
 # Releasing frontend modules
 
-After meeting a reasonable threshold of commits, or after a set time, you will likely want to publish your software to the NPM registry so that consumers can get your latest stable versions.
+[Releasing](https://docs.github.com/en/repositories/releasing-projects-on-github/about-releases) frontend modules is a critical part of the development process. After meeting a reasonable threshold of commits, or after a set amount of time, you will likely want to publish your software to the NPM registry so that consumers can install it. This guide will walk you through the process of releasing frontend modules in the O3 ecosystem.
+
+<Callout type="info">
+  O3 currently follows a release cadence of two weeks. This means we aim to publish new versions of our frontend to our
```

Comment:
I've elected to go with 'two weeks' here, which seems very unambiguous to me, @ibacher.